tests/server.unit.test.js

```diff
@@ -210,6 +211,16 @@ describe('given a server', function () {
     })
   })
 
+  describe('#attach', function () {
+    it('returns ready promise', function () {
+      var httpServer = require('http').createServer(function () {})
+      var radarServer = new Server()
+
+      var returned = radarServer.attach(httpServer, common.configuration)
+      expect(returned).to.equal(radarServer.ready)
```

Comment:
Added a simple test for this- it's not doing a *whole* lot, but it was cheap and may help catch regressions.